tissue-clicker/src/App.js

```diff
@@ -4,27 +4,30 @@ import Home from './Home';
 import Rollers from './Rollers';
 import Decor from './Decor';
 import About from './About';
+import { ClickProvider } from './ClickContext';
 
 function App() {
   return (
-    <Router>
-      <div className="App">
-        <nav>
-          <ul>
-            <li><Link to="/">Home</Link></li>
-            <li><Link to="/rollers">Rollers</Link></li>
-            <li><Link to="/decor">Decor</Link></li>
-            <li><Link to="/about">About</Link></li>
-          </ul>
-        </nav>
-        <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path="/rollers" element={<Rollers />} />
-          <Route path="/decor" element={<Decor />} />
-          <Route path="/about" element={<About />} />
-        </Routes>
-      </div>
-    </Router>
+    <ClickProvider>
```

Comment:
Had to add this here and import ClickContext/Provider so the app knows to hold onto the click count